5index.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+// task 1:
+
+function durationBetweenDates(a,b,c) {
+
+    const a2 = new Date(a);
+    const b2 = new Date(b);
+
+    if (c === 'seconds') console.log((Math.abs((a2 - b2) / 1000)).toString());
```

Comment:
```suggestion
    if (c === 'seconds') {
      console.log((Math.abs((a2 - b2) / 1000)).toString());
    }
```
Відформатуйте код, і з логічними конструкціями завжди видяйте блок коду, а не в один рядок. Так краще читаємість і менше потенційних багів.

Reply:
Тут і далі